app/src/main/java/com/mapbox/maps/testapp/examples/markersandcallouts/infowindow/MarkerManager.kt

```diff
@@ -111,7 +111,7 @@ class MarkerManager(
         offsetX(0)
       }
     )
-    view.visibility = View.GONE
+    view.visibility = View.INVISIBLE
```

Comment:
Actually a bit suprising this doesn't work since docs clearly state that `Interface definition for a callback to be invoked when the global layout state or the visibility of views within the view tree changes. ` [here](https://developer.android.com/reference/android/view/ViewTreeObserver.OnGlobalLayoutListener)

Reply:
well, I guess the docs are not precisely correct then... that happens 😄

Reply:
Or this is actually a bug of Android that we can report to Google 😉

Reply:
Would be nice to report, agree

Reply:
Also - couldn't we instead if listening for `onDraw` keep using GONE? Seems we haven't been able to find broken cases with it?

Reply:
@yunikkk what do you mean? We're not controlling that - users may hide their views (mark them `View.INVISIBLE`) and then make them visible again and expect everything works as expected. Now that use-case is completely broken and that's what we're trying to fix.
I'm reverting L114 to `INVISIBLE` instead of `GONE` because that's how that this example was designed back in time and to showcase that it's working again now as expected.

Reply:
created an issue https://issuetracker.google.com/issues/244067928

Reply:
Shall we also add some inline docs to this example to add more context and suggest to use INVISIBLE instead of GONE?